src/test/java/calculator/CalculatorTest.java

```diff
@@ -0,0 +1,74 @@
+package calculator;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class CalculatorTest {
+
+    private Calculator calculator;
+
+    // 전체 생성하는거 추가
+    @BeforeEach
+    void setCalculator() {
+        calculator = new Calculator();
+    }
```

Comment:
```suggestion
public class CalculatorTest {

    private Calculator = new Calculator();
```

계산기의 상태가 변경되지 않음을 확신할 수 있으므로, 매 테스트마다 새로 인스턴스를 할당해주기보다
딱 1회만 할당해줘도 괜찮을거 같아요 😉

Reply:
아 그러네요. 테스트 전에 만들어야 된다는 생각만 했는데 굳이 BeforeEach까지 쓸필요도 없어보입니다